docs/dashboard/user-guide/self-reporting.md

```diff
@@ -0,0 +1,69 @@
+# Self Report <since project="skills-service" version="1.4.0" />
+
+Self Report is a feature that empowers users to mark skills as completed directly in the SkillTree dashboard OR through the embedded Skills Display component. 
+A project administrator can enable ``Self Reporting`` for a skill, set of skills or even all the skills. 
+Skills that have been configured with Self Reporting display ``I did it`` button allowing users to self report completion of those skills. 
+
+There are two ``Self report`` types available:  
+1. ``Honor System`` - points are awarded immediately 
+1. ``Approval Queue`` - request goes into the project's approval queue; project administrators can approve or deny requests
+
+::: tip 
+Project administrators can craft training profiles consisting of:
+  - only self-reported skills *OR*
+  - a mix of self-reported skills and skills that are reported programmatically *OR* 
+  - a project could have no self-reported skills at all
+:::  
+    
+## Configuring
+
+Self reporting is enabled and configured for each skill individually. 
+When creating or editing a skill 
+1. select ``Self Reporting`` checkbox
+1. then select ``Self Reporting`` type (``Approval Queue`` or ``Honor System``)
+
+::: tip
+By default, Self Reporting is disabled when creating or modifying a skill. 
+If your project primarily consists of Self Reported skills then you can easily change the default by 
+navigation to the ``Project -> Settings`` tab. Then you can enable Self Reporting and select its default type for all the skills that will be created after that point. 
```

Comment:
~~by navigation to the~~ ```by navigating to the```.
~~Then you can...~~ ```There you can enable Self Reporting as the default for all skills created after that point```

Reply:
done